source/automem/ref_counted.d

```diff
@@ -90,8 +92,8 @@ private:
 
         static if(is(Type == class)) {
 
+            align (classInstanceAlignment!Type)
```

Comment:
Basically void[N] is an array of bytes, with `align` we make sure it's aligned properly. And that's my mistake actually. It should be aligned to (void*).alignof instead. I'll update that part.